aws/resource_aws_directory_service_directory_test.go

```diff
@@ -578,24 +657,28 @@ resource "aws_vpc" "main" {
 
 resource "aws_subnet" "foo" {
   vpc_id = "${aws_vpc.main.id}"
-  availability_zone = "us-west-2a"
+  availability_zone = "${data.aws_availability_zones.available.names[0]}"
   cidr_block = "10.0.1.0/24"
   tags = {
     Name = "tf-acc-directory-service-directory-microsoft-foo"
   }
 }
 resource "aws_subnet" "bar" {
   vpc_id = "${aws_vpc.main.id}"
-  availability_zone = "us-west-2b"
+  availability_zone = "${data.aws_availability_zones.available.names[1]}"
   cidr_block = "10.0.2.0/24"
   tags = {
     Name = "tf-acc-directory-service-directory-microsoft-bar"
   }
 }
 `
 
-var randomInteger = acctest.RandInt()
-var testAccDirectoryServiceDirectoryConfig_withAlias = fmt.Sprintf(`
+func testAccDirectoryServiceDirectoryConfig_withAlias(randomInteger int) string {
```

Comment:
Nit: `randomInteger` does not provide context around this arguments expected usage. Nor is it's actual usage easy to spot `alias = "tf-d-%d"`. I would suggest something like `randomAliasSuffix` or `aliasSuffix` to help the reader understand the argument's purpose.

Reply:
This was just a lift and shift of the variable from two lines above, but I will rename as requested.

Reply:
Updated and verified in 5e00cdc473b835fa579333d0c5d151e39c964dd0

Reply:
Thanks - I only called it out because I couldn't find where it was being used without having to scan all of the test configs. :+1: